core/src/main/scala/org/apache/spark/deploy/DeployMessage.scala

```diff
@@ -188,7 +188,7 @@ private[deploy] object DeployMessages {
   }
 
   case class ExecutorUpdated(id: Int, state: ExecutorState, message: Option[String],
-    exitStatus: Option[Int], workerLost: Boolean)
+    exitStatus: Option[Int], workerHost: Option[String])
```

Comment:
We should add some comments to explain what `workerHost` indicates here.

Reply:
and how about  `hostOfLostWorker`?

Reply:
In case of the decommission, the `Worker` doesn't really lose at the time the Master sends `ExecutorUpdated` to the driver. So I think it might not be such accurate.